src/main/java/de/tum/in/www1/artemis/service/compass/CompassCalculationEngine.java

```diff
@@ -118,17 +128,21 @@ private void buildModel(long modelSubmissionId, JsonObject jsonModel) {
         }
     }
 
-    private void buildAssessment(ModelingSubmission submission) {
+    /**
+     * Adds the manual assessment of the given submission to Compass so that it can be used for automatic assessments. Additionally, it marks the submission as assessed, i.e. the
+     * submission is not considered when providing a submission for manual assessment to the client.
+     *
+     * @param submission the submission for which the manual assessment is added
+     */
+    private void addManualAssessmentForSubmission(ModelingSubmission submission) {
         UMLClassDiagram model = modelIndex.getModelMap().get(submission.getId());
```

Comment:
I think it's a bit misleading to call it UMLClassDiagram but also model. I think we need to get more consistent here. What exactly is this UMLClassDiagram?